web/handlers/admin/handler.go

```diff
@@ -32,6 +33,9 @@ func Handler(r *render.Renderer, roomState *roomstate.Manager, al admindb.AllowL
 			Count   int
 		}{lst, len(lst)}, nil
 	}))
+	mux.HandleFunc("/menu", r.HTML("admin/menu.tmpl", func(w http.ResponseWriter, req *http.Request) (interface{}, error) {
```

Comment:
Technically this shouldn't be here. This should only make sense if we were using iframes to embed the template in the page and it should be possible without. The block/template system should be able to do this.

Reply:
Maybe i'm missing something and i'ts an intended feature to _just_ have the menu?

Reply:
Yeah the intention is to have a screen or page that only shows the menu, because you asked for it via the menu button.

We can do this in JS and then we won't need the new route /admin/menu but we want to do things without JS right?

Reply:
oooooh, sorry. I was just wondering if we can lay it out as a list with items next to each other compared to over and under. I didn't think of a separate page per se but that could work, too.

I know people that would be glad with a JS-less page but personally I'm fine with..what's the term.. progressive enhancement? (it should still work without)

Reply:
>  I was just wondering if we can lay it out as a list with items next to each other compared to over and under.

That would work now, but as soon as we add more items to the list (edit notices, manage bans, manage room settings) then it won't fit side-by-side on the top anymore. Also a menu button is quite common in these cases

Reply:
My wish with not having JS is just to keep the codebase simple, easier to maintain and reason about when modifying the UI.